server/fileupdater.mjs

```diff
@@ -1,4 +1,4 @@
-export const VERSION = 12;
+export const VERSION = 50;
```

Comment:
Version 50 already? Avoiding conflicts? I assume we'll change this before merging.

Reply:
yes. I expect this PR will take some time and other versions will come before it. it ill be changed.